File: KekaForum.Models/Question.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace KekaForum.Models.Core
{
    public class Question
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public DateTime DateCreated { get; set; }
        public string UserFirstName { get; set; }
        public string UserLastName { get; set; }
        public string UserProfilePicUrl { get; set; }
        public int UpvotesCount { get; set; }
        public int ViewsCount { get; set; }
        public int AnswersCount { get; set; }
        public Boolean IsResolved { get; set; }
    }
}

File: KekaForum/ClientApp/src/models/Question.ts
export class Question {

    id:number;
    title:string;
    description:string;
    dateCreated:string;
    userFirstName:string;
    userLastName:string;
    userProfilePicUrl:string;
    upvotesCount:number;
    viewsCount:number;
    answersCount:number;
    isResolved:boolean;

    constructor(args:any){
        this.id = args.id;
        this.title = args.title;
        this.description = args.description;
        this.userFirstName = args.userFirstName;
        this.userLastName = args.userLastName;
        this.upvotesCount = args.upvotesCount;
        this.answersCount = args.answersCount;
        this.isResolved = args.isResolved;
    }
}
